kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -778,20 +770,11 @@ private KieScannerStatus mapScannerStatus(KieContainerInstanceImpl kieContainerI
                 messages.add(new Message(Severity.INFO, "Release id successfully updated for container " + id));
                 return new ServiceResponse<ReleaseId>(ServiceResponse.ResponseType.SUCCESS, "Release id successfully updated.", kci.getResource().getReleaseId());
             } else {
-                // no container yet, attempt to create it
```

Comment:
@karreiro I guess that the test failures are related to this change. Do you have some specific reason why did you delete these lines?

Reply:
Hi @stuaakar,

You're absolutely right. The tests are failing because these lines were removed.

However, I think that we should remove the two failing tests:
```
KieServerContainerCRUDIntegrationTest.testUpdateReleaseIdForNotExistingContainer
KieControllerManagementIntegrationTest.testStartContainerByUpdateKieContainerStatus
```

These lines don't start the container properly, and it should be started before the updating process. Of course, I can rollback and the tests will pass again, but it sounds like an invalid behavior.

WDYT? cc @mswiderski

Reply:
@karreiro The question is what should happen if Kie server receives update release id request for a non existing container. Current implementation creates a new container with updated release id.

Reply:
@sutaakar the `KieServerImpl` class is not able to create the container properly during the update process.

If we really want to create a container in the `updateContainerReleaseId`, we would need to change signature from this:
```
updateContainerReleaseId(String id, ReleaseId releaseId)
```
to something like this:
```
ServiceResponse<ReleaseId> updateContainerReleaseId(String containerId, KieContainerResource container);
```
But, I had a chat with @mswiderski, and we decided that we should keep the API as is and not start the container during the update process.

To keep the old behavior I'm starting the container _before_ the upgrade process, see:
https://github.com/karreiro/droolsjbpm-integration/blob/14426be667e4d1e20749bac15f4ab46478f5ddd6/kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java#L322

Reply:
@karreiro sounds good